spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'requires good passwords' do
+    it 'accepts good passwords' do
+      expect { FactoryBot.create :user, password: 'P@ssw0rd}' }.not_to raise_error
+    end
+    it 'rejects blank passwords' do
+      expect { FactoryBot.create :user, password: '' }.to raise_error(ActiveRecord::RecordInvalid)
+    end
```

Comment:
There might be an additional test here for < 8 chars, but also not blank, but this suffices for sure.

Reply:
Added one. thanks for pointing it out :-D